content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -147,6 +147,10 @@ Secrets.
 - https://kubernetes.io/docs/concepts/configuration/secret/#decoding-a-secret
 - https://kubernetes.io/docs/concepts/configuration/secret/#secret-and-pod-lifetime-interaction
 
+Once the password has been changed and the pods have come up in a healthy state, you'll need the application to use the updated VCAP service.  Typically, you'll unbind the service from the application (`cf unbind-service`), bind it again (`cf bind-service`), then restage the application (`cf restage`)
```

Comment:
I'd removed the `typically` here. These are the only clear instructions for getting the broker to report the correct credentials to the application instance.

1. Unbind the service to the app
1. Bind the service to the app
1. Restage the application